funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -687,7 +689,9 @@ def start(self, poll_period=None):
                             msg['parsl_v'] != self.current_platform['parsl_v']):
                             logger.warn("[MAIN] Manager {} has incompatible version info with the interchange".format(manager))
 
-                            if self.suppress_failure is False:
+                            if self.relax_manager_version_match is True:
+                                logger.info("[MAIN] Ignoring version match requirements and continuing")
```

Comment:
This logging (here and line 690) needs tidying: if we're expecting a user to be allowed to run different manager and interchange versions, we shouldn't be warning them that they're incompatible and talking about a version match "requirement" that is not a requirement.
If this is a supported feature, this is debug or info level information and should not be phrased this way.

Use "different" rather than "incompatible" ?

In line 691 below tell the user kill event is being set because actually in this code path it *is* a requirement that version match, unlike the default behaviour.

Reply:
I've updated the text here to read:

```
logger.info(f"[MAIN] Manager:{manager} version:{msg['python_v']} does not match the interchange")
```